integration-tests/src/test/scala/play/libs/ws/ahc/AhcWSClientSpec.scala

```diff
@@ -99,10 +99,11 @@ class AhcWSClientSpec(implicit executionEnv: ExecutionEnv) extends Specification
         val responseXml = response.getBody(xml())
         responseXml.normalizeDocument()
 
-        responseXml.isEqualNode(document) must beTrue
+        responseXml.isEqualNode(document) must beTrue and {
```

Comment:
I don't think so, as for other methods depending on the underlying AHC which are only tested there.